hoodie-client/src/main/java/com/uber/hoodie/io/HoodieCommitArchiveLog.java

```diff
@@ -173,6 +189,49 @@ private boolean deleteArchivedInstants(List<HoodieInstant> archivedInstants) {
         throw new HoodieIOException("Failed to delete archived instant " + archivedInstant, e);
       }
     }
+
+    // Remove older meta-data from auxiliary path too
+    Optional<HoodieInstant> youngestCommitted =
+        archivedInstants.stream()
+            .filter(i -> {
+              return i.isCompleted()
+                  && (i.getAction().equals(COMMIT_ACTION) || (i.getAction().equals(DELTA_COMMIT_ACTION)));
```

Comment:
the inclusion of delta_commit_action also is to facilitate move of delta commit/rollback metadata?

Reply:
Yes